spec.bs

```diff
@@ -8,38 +8,40 @@ Repository: dylanvanassche/rdf-ts-snippets-spec
 URL: https://dylanvanassche.github.io/rdf-ts-snippets-spec/
 Editor: Gerald Haesendonck
 Editor: Dylan Van Assche
-Abstract: A RDF Time Series Snippet is a subset of a Time Series described in RDF. [=Snippet=]s aim to be more compact compared to RDFS Collection and Containers for describing Time Series by levering the Common Data Types.
+Editor: [Pieter Colpaert](https://pietercolpaert.be/#me)
+Abstract: This document specifies how time series data with with only a few changing values
+  can be described in RDF.
 Markup Shorthands: markdown yes, css no
 </pre>
 
-# Introduction # {#intro}
+# Abstract # {#abstract}
 
-Sensor measurements, positioning data, etc. are commonly published as [=Time Series=]
-data with a timestamp and a value. 
-Representing [=Time Series=] data in RDF drastically raises the verbosity,
-which Time Series Snippets try to address by compacting the data points in subsets of a [=Time Series=]
-through [Common Data Types](https://w3id.org/awslabs/neptune/SPARQL-CDTs/spec/latest.html) such as `cdt:List` and `cdt:Map`.
+An RDF Time Series Snippet is a segment of data points from a time series dataset, typically used for analysis or visualization.
+The specification uses [SPARQL Common Data Types](https://awslabs.github.io/SPARQL-CDTs/spec/latest.html) coined by AWS Lab to encode the data points in a values literal.
+The entity can then be used for annotating this snippet, which is a time interval subset of a potentially larger time series.
```

Comment:
Shouldn't we need to keep using `[=Time Series=]`? Bikeshed will point then to the definition we have here in the spec.

Reply:
I mark it once, but it applies to all definitions across the whole PR.